README.md

```diff
@@ -33,16 +76,49 @@ This project is to test our ability to store data and delete them, to make a lis
 - Twitter: [@preateamie](https://twitter.com/nualart)
 - LinkedIn: [PriscillaOkoro](www.linkedin.com/in/cnualart)
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
 
-## 🤝 Contributing
+<p align="right">(<a href="#awesome books">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-## Show your support
+Feel free to check the [issues page](https://github.com/Prisca-tech/AWESOME_BOOKS/issues).
+
+<p align="right">(<a href="#awesome books">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please visit the repository and give it a star ⭐️ 
+
+<p align="right">(<a href="#awesome_books">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank my partner for working tirelessly on this project
+
+<p align="right">(<a href="#awesome books">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] KIndly fix the MIT License on your readme file. It is broken at the moment.

### Error 404
![license](https://github.com/Prisca-tech/AWESOME_BOOKS/assets/26597432/c3529a7c-ba8d-4da7-9e17-2eda9af85752)